AccountService.js

```diff
@@ -5,6 +5,11 @@
 
 import axios from 'axios';
 const headers = {Accept: 'application/ld+json, application/json'};
+/**
+ * @class AccountService
+ * @description this service is used to encapsulate all account
```

Comment:
I think we should prefer the style of having the description as the text right at the top.  In theory it'll use the first line as description, or first line as summary and further text as longer description.
```js
/**
 * Blah blah.
 *
 * Extended blah blah blah blah blah blah.
 * 
 * @param ... foo ...
 */
... blahblah(foo) ...
```

Reply:
- [ ] using the initial comment as the description.